jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -610,7 +612,9 @@ YUI.add('deployment-flow', function() {
     },
 
     render: function() {
-      var disabled = this.props.acl.isReadOnly();
+      var disabled = this.props.acl.isReadOnly() ||
```

Comment:
this.state.deploying is already a boolean, so === true is not required, this could be rewritten as
```javascript
const disabled = this.props.acl.isReadOnly() || this.state.deploying;
```